2-cloud-sql/Gemfile

```diff
@@ -29,7 +29,8 @@ end
 gem "tzinfo-data"
 
 group :test do
-  gem "rspec-rails"
+  gem "rspec-rails", '3.5.0.beta3'
```

Comment:
this is so we can use the `config.when_first_matching_example_defined(:e2e)` function in rspec, which works incredibly well for e2e test deployment. I'll be adding these to the other steps as well.